circleci/images/debbuilder/files/install-builddeps

```diff
@@ -23,17 +23,23 @@ echo "Writing /etc/apt/sources.list.d/pgdg.list..." >&2
 cat > /etc/apt/sources.list.d/pgdg.list <<EOF
 deb     http://deb.debian.org/debian ${codename}-backports                main
 deb-src http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg         main
-deb-src http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg-testing 12
```

Comment:
in `circleci/images/failtester/files/sbin/install-testdeps` we define a variable; `pg_latest`, could we do so here as well? It seems the number is just indicating the latest pg version.

Ideally we would pass this in via build args to the docker build so to not have to keep editing these files for every major pg release. That is, as long as the build doesn't change like it did this time.